src/reclaim_python_sdk/utils/session_utils.py

```
@@ -66,4 +66,31 @@ async def update_session(session_id, status):
+        raise UpdateSessionError(f'Error updating session with sessionId: {session_id}')
+
+async def fetch_status_url(session_id: str) -> StatusUrlResponse:
+    from .errors import StatusUrlError
+    from .types import StatusUrlResponse
+    
+    validate_function_params([
+        {'input': session_id, 'param_name': 'sessionId', 'is_string': True}
+    ], 'fetch_status_url')
+
+    try:
+        response = requests.get(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)